server/tests/unit/clientmanager.spec.ts

```diff
@@ -172,3 +172,15 @@ describe("BalancerManager", () => {
 		expect(balancerManager.balancerConnections).not.toContain(con2);
 	});
 });
+
+describe("MonolithId", () => {
+	it("should have unique IDS for each instance of a monolith", () => {
```

Comment:
From this context, there is only 1 monolith: the one that is running. The client manager should send the same id to both of these mock connections.